tree/ntuple/v7/doc/BinaryFormatSpecification.md

```diff
@@ -1059,6 +1059,17 @@ The limits refer to a single RNTuple and do not consider combinations/joins such
 | Maximum string length (meta-data)              | 4GB                          | String encoding                                        |
 | Maximum RBlob size                             | 128 PiB                      | 1GiB / 8B * 1GiB (with maxKeySize=1GiB, offsetSize=8B) |
 
+## Naming specification
+
+The allowed character set for the strings representing the name of an RNTuple, as well as the name of a field, is
+restricted to Unicode characters encoded as UTF-8, with the following exceptions:
+
+* All control codes. These notably include newline (U+000A) and horizontal tab (U+0009).
+* Full stop (U+002E '&#46;')
```

Comment:
This is the UK formulation (`Period` is the US formulation).

Reply:
Thanks, I just took it from https://en.wikipedia.org/wiki/List_of_Unicode_characters, in any case the Unicode code disambiguates :+1: